content/docs/user-guide/contributing/docs.md

```diff
@@ -140,8 +149,8 @@ Some available variables:
 
 ## Doc style guidelines (JavaScript and Markdown)
 
-Some the rules below are be applied automatically by a pre-commit Git hook that
-is installed when `yarn` runs (see [dev env](#development-environment)).
+Some of the following rules are applied automatically by a pre-commit Git hook that is
+installed when `yarn` runs (see [dev env](#development-environment)).
```

Comment:
Restyled didn't catch this for some reason... 😕 

```suggestion
Some of the following rules are applied automatically by a pre-commit Git hook
that is installed when `yarn` runs (see [dev env](#development-environment)).
```

Reply:
Somehow my VSCode prettier and yarn's prettier behave differently sometimes, in minor ways. There is also some discrepancy between Git's handling of CRLF and Prettier's config. It should be OK in my latest commit.

Reply:
Maybe related: https://github.com/iterative/dvc.org/issues/2641